### PR TITLE
Suse fixes

### PR DESCRIPTION
**Pull request by dev-zero**

### CODE REVIEW

**Comment by RichardM99** [comment]
Any plans for this PR to be merged?

File: templates/timezone.j2 [hunk 1]
```diff
@@ -1 +0,0 @@
-{{ ntp_timezone }}
```
Comment (geerlingguy) [inline templates/timezone.j2]: Is this template used anywhere?

Comment (dev-zero) [reply templates/timezone.j2]: no, according to history it was used in `main.yml` for Debian at some point. This is why I propose to remove it.

Comment (geerlingguy) [reply templates/timezone.j2]: Okay thanks!

**Pull request merged**